source/deploy/deployed-systems.rst

```diff
@@ -0,0 +1,199 @@
+Deployed Architecture
+=====================
+
+Let's start with an overview of some typical architectures for
+our deployed sites.
+
+Note that depending on the expected
+traffic of a site, we might run everything on one
+server, or spread services out onto different servers
+(e.g. web servers, celery workers, database master,
+cache, etc.).
+
+Operating system
+~~~~~~~~~~~~~~~~
+
+Our server operating system of choice is Ubuntu Server LTS.
+For new projects, we'll use the most recent LTS release.
+
+We won't necessarily update older projects that might be on
+an older LTS release, as long as they're still getting
+security updates and we don't need some feature that's only
+available on a newer release.
+
+System updates
+--------------
+
+We have started enabling automated updates using Ubuntu's
```

Comment:
Technically `unattended_upgrades` is currently an optional feature, since it's not in the default project template `top.sls` file. So maybe change the first sentence to something like "Ubuntu's automated updates can be enabled by adding `unattended_upgrades` to your top.sls file."